### ISSUE TITLE
ESP32 WROOM Wifi_manager problem

### ISSUE THREAD

**Issue by Andreafoxalarm** [tzapu/WiFiManager#776]
Good evening to everyone
I'm using the wifi manager on esp32.
Everything works fine as soon as I program my module, but once I make the first connection to a wifi network, if I try to connect the module to another network, the module starts with autoconnect but if I click on the configure wifi button I do not see any network. I attach an image

![screenshot_20181129-192352_samsung internet](https://user-images.githubusercontent.com/44032260/49242958-6b31a280-f40c-11e8-9ed6-bc04388c6752.jpg)

thank you all

**Comment by tablatronix** [comment]
Are you using development branch? It doesnt look like it

**Comment by Andreafoxalarm** [comment]
I'm using the esp32 wroom module on a DEVKITV1.
Initially the wifi manager works, but after connected to a network when I try to change from my phone even pressing the scan button I see no network anymore.
Forgive me, but I am not an arduino expert

**Comment by tablatronix** [comment]
what version of wm are you using

**Comment by dragondaud** [comment]
I can attest that using https://github.com/espressif/arduino-esp32 core and https://github.com/tzapu/WiFiManager/tree/development works fine for me on WROOM32.

**Comment by tablatronix** [comment]
Yeah they are clearly using some non official esp32 fork from the pics

**Comment by Andreafoxalarm** [comment]
Now download the files from your link and try again.
maybe I fell in error and I was wrong.
thank you all for your help

**Comment by Gior80** [comment]
I'm having exactly same problem.  getting an event mismatch ,  when scanning get absolutely nothing ! tried both official release and development branch

*WM: 192.168.4.1
*WM: HTTP server started
E (89005) event: mismatch or invalid event, id=63
E (89006) event: default event handler failed!
E (98162) event: mismatch or invalid event, id=63
E (98162) event: default event handler failed!
dhcps: send_nak>>udp_sendto result 0
E (112349) event: mismatch or invalid event, id=63
E (112349) event: default event handler failed!
dhcps: send_offer>>udp_sendto result 0
*WM: Request redirected to captive portal
*WM: Handle root
*WM: Request redirected to captive portal
*WM: Handle root
*WM: Request redirected to captive portal
*WM: Handle root
*WM: Scan done

**Comment by tablatronix** [comment]
thats interesting and everything works you just dont get a wifi list ?

**Comment by tablatronix** [comment]
Esp bug should be fixed soon
https://github.com/espressif/esp-idf/issues/2916

**Comment by tablatronix** [comment]
Can you tell me which example or code you are using?

**Comment by aurelijusrozenas** [comment]
I don't know if this will be helpful but I got same error `event: mismatch or invalid event, id=63` using [Basic.in](https://github.com/tzapu/WiFiManager/blob/development/examples/Basic/Basic.ino) example. The problem went away when I removed `, "password"` from `res = wm.autoConnect("AutoConnectAP","password"); // password protected ap`.

**Comment by rjwats** [comment]
espressif/esp-idf#2916 appears to be resolved now
